cpp/bench/ann/src/common/benchmark.hpp

```diff
@@ -483,6 +483,14 @@ void register_search(std::shared_ptr<const dataset<T>> dataset,
                   ->MeasureProcessCPUTime()
                   ->UseRealTime();
 
+      if (metric_objective == Mode::kThroughput) {
+        if (index.algo.find("faiss_gpu") != std::string::npos) {
```

Comment:
This is, again, putting algorithm specific stuff in the common files. Please put this in the FAISS benchmark files.

Reply:
Fixed this! Marking as resolved.